dotCMS/src/main/java/com/dotcms/publisher/pusher/wrapper/ContentTypeWrapper.java

```diff
@@ -31,12 +31,12 @@ public void setFieldVariables(List<FieldVariable> fieldVariables) {
         this.fieldVariables = fieldVariables;
     }
 
-	public String getWorkflowSchemaName() {
-        return workflowSchemaName;
+	public List<String> getWorkflowSchemaNames() {
+        return workflowSchemaNames;
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Return a copy of "workflowSchemaNames". [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS2384)